spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -75,7 +118,9 @@
       define_model :child
       @model = define_model :parent do
         has_and_belongs_to_many :children
-      end.new
+      end
+      .new
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression in an assignment spanning multiple lines.